sdk/storage/azure_storage_blob/tests/blob_service_client.rs

```diff
@@ -122,3 +125,46 @@ async fn test_list_containers_with_continuation(ctx: TestContext) -> Result<(),
 
     Ok(())
 }
+
+#[recorded::test]
+async fn test_get_service_stats(ctx: TestContext) -> Result<(), Box<dyn Error>> {
+    //TODO: Need playback only, need secondary-enabled GRS account to record against
+    // Recording Setup
+    let recording = ctx.recording();
+    let service_client = get_blob_service_client(recording)?;
+
+    let response = service_client.get_service_stats(None).await?;
```

Comment:
Seems like this only works against secondary? And in Python these are playback only as the newly spun up account in the live test pipeline does not persist long enough for meaningful logs to generate.